storage/persistent_storage/persistent-storage-vsphere.adoc

```diff
@@ -27,7 +27,7 @@ requested by users.
 Dynamically provisioning VMware vSphere volumes is the recommended method.
 
 .Prerequisites
-An {product-title} cluster installed on a VMware vSphere version that meets the requirements for the components that you use. See xref:../../installing/installing_vsphere/installing-vsphere.adoc[Installing a cluster on vSphere]
+An {product-title} cluster installed on a VMware vSphere version that meets the requirements for the components that you use. See xref:../../installing/installing_vsphere/installing-vsphere.adoc[Installing a cluster on vSphere].
```

Comment:
@qinpingli Can you PTAL at this prereq that I added to verify that the link is working and that the page it points to makes sense to you? 

For more background info, the BZ asks to "add a section specifying the supported versions of VMWare vSphere", but that information can potentially change (e.g., vSphere 6.7U2 vs. 6.7U3), as evidenced in [PR 19866](https://github.com/openshift/openshift-docs/pull/19866). So it makes the most sense to link to that vSphere reqs page, rather than adding the section here in vSphere storage. Thanks!

Reply:
@bobfuru LGTM